### PR TITLE
Updates to Microsoft_Teams_URL_Analysis

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description: 

This makes some general updates to the Microsoft_Teams_URL_Analysis flow. It adds proper output formatting and an error message for VT errors. 

## Testing: 
Was imported to ICON and tested via teams. 

See updated screen shots for tests.

### CODE REVIEW

File: workflows/Microsoft_Teams_URL_Analysis/Microsoft_Teams_URL_Analysis.icon [hunk 1]
```diff
@@ -209,70 +221,202 @@
...
-              "defaultEdgeId": "6ed175a8-2f22-48bf-8e22-ee29714efd4e",
-              "stepControlParams": [
-                {
-                  "edgeId": "6ed175a8-2f22-48bf-8e22-ee29714efd4e",
-                  "expression": {},
-                  "expressionText": ""
+              "input": {
+                "channel_name": "Komand Test Channel",
+                "message_content": "<b>📃  Could not get VirusTotal report  📃</b><hr>\n\nThis may happen if you overload the VirusTotal API with requests. Please wait a couple of minutes and try again.<br> \n<br>\nMake sure no other workflows are hitting the VirusTotal API excessively. <br>\n<br>\n<b>For more information:</b><br>\n<a href={{[$job].[URL]}}>Job Report</a>",
+                "team_name": "Komand-Test-Everyone"
```
Comment (mrinehart-r7) [inline workflows/Microsoft_Teams_URL_Analysis/Microsoft_Teams_URL_Analysis.icon:242]:
```suggestion
                "team_name": "changeme"
```

File: workflows/Microsoft_Teams_URL_Analysis/Microsoft_Teams_URL_Analysis.icon [hunk 2]
```diff
@@ -209,70 +221,202 @@
...
             "isDisabled": false,
             "parameters": {
-              "defaultEdgeId": "6ed175a8-2f22-48bf-8e22-ee29714efd4e",
-              "stepControlParams": [
-                {
-                  "edgeId": "6ed175a8-2f22-48bf-8e22-ee29714efd4e",
-                  "expression": {},
-                  "expressionText": ""
+              "input": {
+                "channel_name": "Komand Test Channel",
```
Comment (mrinehart-r7) [inline workflows/Microsoft_Teams_URL_Analysis/Microsoft_Teams_URL_Analysis.icon:240]:
```suggestion
                "channel_name": "changeme",
```

File: workflows/Microsoft_Teams_URL_Analysis/Microsoft_Teams_URL_Analysis.icon [hunk 3]
```diff
@@ -417,6 +561,214 @@
...
+              "slugName": "microsoft_teams",
+              "slugVersion": "2.0.2",
+              "imageData": "https://us.cdn-assets.connect.insight.rapid7.com/icons/rapid7/microsoft_teams/2.0.2/icon.png"
+            },
+            "identifier": "send_html_message",
+            "continueOnFailure": false,
+            "isDisabled": false,
+            "parameters": {
+              "input": {
+                "channel_name": "Komand Test Channel",
```
Comment (mrinehart-r7) [inline workflows/Microsoft_Teams_URL_Analysis/Microsoft_Teams_URL_Analysis.icon:580]:
```suggestion
                "channel_name": "changeme",
```

File: workflows/Microsoft_Teams_URL_Analysis/Microsoft_Teams_URL_Analysis.icon [hunk 4]
```diff
@@ -417,6 +561,214 @@
...
+                "message_content": "<b>📃 Starting URL Anaysis 📃</b><hr>\nURL: {{[11d22bb8-eda1-479d-820b-c99f67b811b1].[trimmed]}}",
+                "team_name": "Komand-Test-Everyone"
```
Comment (mrinehart-r7) [inline workflows/Microsoft_Teams_URL_Analysis/Microsoft_Teams_URL_Analysis.icon:582]:
```suggestion
                "team_name": "changeme"
```

**Comment by jmcadams-r7** [comment]
## Icon Validate: 
```
rmt-mbp-5357:Microsoft_Teams_URL_Analysis jmcadams$ icon-validate .
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 31.09ms
```

**Pull request merged**